packages/client-search/src/__tests__/integration/dictionary.test.ts

```diff
@@ -32,8 +32,13 @@ describe(testSuite.testName, () => {
     const stopwords = await client.searchDictionaryEntries('stopwords', stopwordEntry.objectID);
 
     expect(stopwords.nbHits).toEqual(nbSearchEntries + 1);
-    expect(stopwords.hits[nbSearchEntries].objectID).toEqual(stopwordEntry.objectID);
-    expect(stopwords.hits[nbSearchEntries].word).toEqual(stopwordEntry.word);
+    expect(stopwords.hits).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({
+          ...stopwordEntry,
+        }),
```

Comment:
```suggestion
        expect.objectContaining(stopwordEntry),
```

this is essentially the same